src/pykeen/ablation/ablation.py

```diff
@@ -6,29 +6,81 @@
 import json
 import logging
 import os
+import sys
+import time
 from copy import deepcopy
-from typing import Any, Mapping
+from typing import Any, Mapping, Optional
+from uuid import uuid4
 
 from ..training import _TRAINING_LOOP_SUFFIX
 from ..utils import normalize_string
 
 __all__ = [
+    'ablation_pipeline',
     'prepare_ablation_from_config',
     'prepare_ablation',
 ]
 
 logger = logging.getLogger(__name__)
 
 
+def ablation_pipeline(
+    config: Mapping[str, Any],
+    directory: Optional[str],
+    dry_run: bool,
+    best_replicates: int,
+    save_artifacts: bool,
+    move_to_cpu: bool,
+    discard_replicates: bool,
+) -> None:
+    """Generate a set of HPO configurations.
```

Comment:
@mali-git also need better docs here (and other functions in this file)

Reply:
Done in 78971d6bd839176e9c1dc758a9b5c9b0e2fd08e3.